app/main.py

```diff
@@ -1 +1,68 @@
-# write your code here
+class SoftwareEngineer:
+    skills = []
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    @classmethod
+    def learn_skill(cls, skill: str):
+        cls.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super(FrontendDeveloper, self).__init__(name=name)
```

Comment:
there is no need to specify "Classname, self" in super() statement, but it is up to u

Reply:
thank you for your review!